src/assets/styles/50-utilities/_sticky.scss

```diff
@@ -0,0 +1,11 @@
+// ----------------------------------
```

Comment:
As we're not using this yet I think we're adding unnecessary bloat to the styles. Should we save it for a future PR? What do you think?